src/styles/index.scss

```diff
@@ -1,3 +1,23 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-color: #00acdc;
+$button-text-color: #fff;
+
+@import  "./card.scss";
+@import  "./stars.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  padding: 0;
   margin: 0;
 }
```

Comment:
Consider moving general styling to a separate file and keeping the main file clean